bin/uploadBrowser.js

```diff
@@ -73,34 +73,34 @@ var OS_IDENTIFIER = 2
 var recipes = [
   // Linux
   ['dist/Brave.tar.bz2', 'multi-channel/releases/CHANNEL/VERSION/linux64', 'linux'],
-  ['dist/brave_VERSION_amd64.deb', 'multi-channel/releases/CHANNEL/VERSION/debian64', 'linux'],
-  ['dist/brave-VERSION.x86_64.rpm', 'multi-channel/releases/CHANNEL/VERSION/fedora64', 'linux'],
+  ['dist/brave{{channelName}}_VERSION_amd64.deb', 'multi-channel/releases/CHANNEL/VERSION/debian64', 'linux'],
```

Comment:
I used a [handlebars.js type of token](http://handlebarsjs.com/) since it's easy to parse out and it's pretty widely recognized

Reply:
maybe we can upgrade this pattern to tools/buildPackages.js and tools/buildInstaller.js too